tinygrad/runtime/ops_metal.py

```diff
@@ -11,13 +10,17 @@
 
 class _METAL:
   def __init__(self):
-    self.mtl_buffers_in_flight: List[Any] = []
     self.device = Metal.MTLCreateSystemDefaultDevice()
+    self.dispatch_group = libdispatch.dispatch_group_create() 
     self.mtl_queue = self.device.newCommandQueue()
-  # TODO: is there a better way to do this?
+  def command_buffer(self):
+    command_buffer = self.mtl_queue.commandBuffer()
+    libdispatch.dispatch_group_enter(METAL.dispatch_group)
```

Comment:
why is this using `METAL` over `self`?

Reply:
Good catch @dc-dc-dc, this has now been fixed. The code now uses `self.dispatch_group` to better encapsulate the `_METAL` class. Thanks for pointing it out!